Svc/Ccsds/Types/Types.fpp

```diff
@@ -1,10 +1,15 @@
-# ------------------------------------------------
-# CCSDS protocols header and trailer types
-# ------------------------------------------------ 
-
 module Svc {
 module Ccsds {
 
+    @ Enum representing an error during framing/deframing in the CCSDS protocols
+    enum FrameError: U8 {
+        SP_INVALID_LENGTH = 0
+        TC_INVALID_SCID = 1
+        TC_INVALID_LENGTH = 2
```

Comment:
There's two `INVALID_LENGTH`, one per protocol. Because knowing which protocol failed may be useful information. And I didn't want to have one enum per protocol, especially given that SpacePacket only has one failure mode.
Thoughts?